CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 **Added**
 
+* Minor refactoring of the page organisation (`#141 <https://github.com/qbicsoftware/sample-tracking-status-overview/issues/141>`_)
```

Comment:
Since we thankfully are not yet at 141 issues i suspect you meant the pull request link? :) 
```suggestion
* Minor refactoring of the page organisation (`#141 <https://github.com/qbicsoftware/sample-tracking-status-overview/pull/141>`_)
```

Reply:
FYI as I learned it doesnt matter if you have pull or issue. It directs to the correct page.